README.md

```diff
@@ -10,10 +10,16 @@
 [![Build Agent](https://github.com/deislabs/akri/workflows/Build%20Agent/badge.svg?branch=main&event=push)](https://github.com/deislabs/akri/actions?query=workflow%3A%22Build+Agent%22)
 [![Test K3s, Kubernetes, and MicroK8s](https://github.com/deislabs/akri/workflows/Test%20K3s,%20Kubernetes,%20and%20MicroK8s/badge.svg?branch=main&event=push)](https://github.com/deislabs/akri/actions?query=workflow%3A%22Test+K3s%2C+Kubernetes%2C+and+MicroK8s%22)
 
+<img src="art/icon/akri-icon-light.svg" alt="Akri logo" width="100"> 

```

Comment:
would be great to conditionally use the dark/light logo depending on whether people are using dark/light mode. Not sure how to do this, though.

Reply:
just an idea, but, what about using: akri-logo-horizontal-light.svg instead?  you've done what [K8s](https://github.com/kubernetes/kubernetes/blob/master/README.md) did, but i kind of like what [etcd](https://github.com/etcd-io/etcd/blob/master/README.md) did.

Reply:
I was trying to decide between those options, too. I didnt like how [thanos](https://github.com/thanos-io/thanos) removed the title, worring about if the image didn't render, but etcd doing both seems like a good balance. I'm open to changing it

Reply:
i feel like the cube icon just kind of dangles where it is.  i'm not immediately connecting it as the logo.

Reply:
Made the logo and wordmark the title. I think it looks much better. Thoughts?

Reply:
i like it.  my only nit is i wish it were a little smaller.  lol.

Reply:
New updates looks good. +1 on being a bit smaller.